terraform/modules/app/main.tf

```diff
@@ -15,6 +15,7 @@ locals {
   subdomain               = var.service_name
   container_services      = "${length(var.custom_container_services) == 0 ? [{ container_service = "${local.subdomain}", port = 80, protocol = "http" }] : var.custom_container_services}"
   service_security_groups = concat([aws_security_group.service.id], var.extra_security_groups)
+  security_group_name     = "${terraform.workspace == "default" ? var.service_name : "${var.service_name}-${terraform.workspace}"}"
```

Comment:
I think it would be clearer to pass in the security group `name` to the module, then this module doesn't need to know about the workspaces.

```
security_group_name = aws_security_group.publisher.name
```

Reply:
I'm not sure about this one, the good thing with having the interpolation in the module is that this way we don't have to put it in every new apps we will write in the future.